Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabIntegrationManager.cpp

```diff
@@ -720,6 +720,12 @@ namespace AzToolsFramework
                         m_prefabOverridePublicInterface->AreOverridesPresent(selectedEntity))
                     {
                         QAction* revertAction = menu->addAction(QObject::tr("Revert Overrides"));
+                        revertAction->setToolTip(QObject::tr("Revert all overrides on this entity to whatever is in the prefab file."));
```

Comment:
Does it revert to values in the prefab files, or values in the loaded in-memory template (which could be unsaved)?

Reply:
That's a good question. From a user perspective, I would imagine that this goes to the last saved state. @forkercat  / @srikappa-amzn do you know what actually happens ?

Reply:
It reverts the values in memory which also supports undo/redo. Yeah I believe the moment we do the reversion the focused prefab becomes dirty and unsaved.

Reply:
So it reverts to the unsaved state that is in memory as opposed to the file on disk ?

Reply:
Is this by design ?

Reply:
I think I might not understand the question correctly.

> Does it revert to values in the prefab files, or values in the loaded in-memory template (which could be unsaved)?

For 'or values in the loaded in-memory template', @michabr do you refer to new and unsaved override edits or template edits?

What revert does is just drop all override patches stored in the current focused prefab on that entity. Values would be reverted to the state what its owning prefab see.

Reply:
Yeah, I wanted to see if we need to reword the tooltip to make it more accurate because reverting the overrides does not restore any values from disk.

Reply:
I think we can keep it short as "Revert all overrides on this entity" and include the "Use delete instead" info.